pkgs/development/tools/misc/universal-ctags/default.nix

```diff
@@ -1,17 +1,17 @@
-{ lib, stdenv, fetchFromGitHub, autoreconfHook, pkg-config, perl, pythonPackages, libiconv, jansson }:
+{ lib, stdenv, fetchFromGitHub, autoreconfHook, pkg-config, perl, python3Packages, libiconv, jansson }:
 
-stdenv.mkDerivation {
+stdenv.mkDerivation rec {
   pname = "universal-ctags";
-  version = "unstable-2019-07-30";
+  version = "p5.9.20210124.0";
 
   src = fetchFromGitHub {
     owner = "universal-ctags";
     repo = "ctags";
-    rev = "920e7910146915e5cae367bc9f135ffd8b042042";
+    rev = version;
     sha256 = "14n3ix77rkhq6vq6kspmgjrmm0kg0f8cxikyqdq281sbnfq8bajn";
```

Comment:
It doesn't seem ok that I changed the version without changing sha256 sum and everything worked.

Reply:
I looked at https://discourse.nixos.org/t/troubleshooting-fetch-commands-like-fetchfromgithub/6159 and it's expected.